src/components/Blocking/BlockPostAnonymous.js

```diff
@@ -5,6 +5,7 @@ import {StyleSheet, Text, View} from 'react-native';
 import ItemListLarge from './ItemListLarge';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import { COLORS } from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from `../../utils/theme` is added, but the old import `{colors} from '../../utils/colors'` is not removed. If `colors` is no longer used in this file, it should be removed to avoid unnecessary imports.
```diff
- import {colors} from '../../utils/colors';
+ import { COLORS } from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@agit26milan masih ada `import {colors} ...` better dihapus aja mas import ini karena tidak dipakai